README.md

```diff
@@ -125,9 +125,9 @@ Let's compose two functions:
 Fn1<Integer, Integer> add = x -> x + 1;
 Fn1<Integer, Integer> subtract = x -> x -1;
 
-Fn1<Integer, Integer> noOp = add.andThen(subtract);
+Fn1<Integer, Integer> noOp = add.fmap(subtract);
 // same as
-Fn1<Integer, Integer> alsoNoOp = subtract.compose(add);
+Fn1<Integer, Integer> alsoNoOp = subtract.fmap(add);
```

Comment:
Whoops, I missed this in the first commit. This `fmap` should be `contraMap` to preserve the same order of operations (`add` then `subtract`) as the example immediately above it. I'll accept the PR and make the change on my end.